Source/Eto.WinForms/WinConversions.cs

```diff
@@ -403,7 +403,7 @@ public static MouseButtons ToEto(this swf.MouseButtons button)
 
 		public static Graphics ToEto(this sd.Graphics g)
 		{
-			return new Graphics(new GraphicsHandler(g));
+			return new Graphics(new GraphicsHandler(g, false));
```

Comment:
I hope we now understand each other, but for sure: We don't want to dispose `sd.Graphics` (as this is the responsibility of the creator of the object = caller of WinForms `OnPaint` - done in `WmPaint` called from `WndProc`)

Original `DrawableHandler.OnPaint` would call `ToEto` and actually mark `sd.Graphics` to be disposed together with `GraphicsHandler` and Eto `Graphics`. Exactly what we **don't** want.

My modification passes `false` to the constructor to correct it (and not dispose `sd.Graphics` which you did not create).